.github/actions/download-translations/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Create Release Numbers'
+description: 'Creates the current release number based on checked out code'
```

Comment:
Only thing that stood out to me, copy and paste? :)

Reply:
I would never.... ;-)